plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerTypeMapping.java

```diff
@@ -91,6 +91,59 @@ public void testReadTime()
                 .addRoundTrip("time(7)", "'23:59:59.9999999'", createTimeType(7), "TIME '23:59:59.9999999'")
 
                 .execute(getQueryRunner(), sqlServerCreateAndInsert("test_time"));
+
+        // inserts need to specify the type in the literal
```

Comment:
That shouldn't be true for `trinoCreateAndInsert` as `io.trino.testing.datatype.CreateAndInsertDataSetup#tableDefinition` infers the types from literals.

isn't the comment a left over since you had `sqlServerCreateAndInsert` here?

Reply:
`CREATE TABLE` is always generated correctly, this is about how values are passed in the `INSERT` statement.

I agree the comment is confusing right now, but I'm not sure how to phrase it better.

Reply:
I checked that this works (removed explicit input and output types)

```
        SqlDataTypeTest.create()
                .addRoundTrip("TIME '00:00:00'", "TIME '00:00:00'")
                .addRoundTrip("TIME '00:00:00.000000'", "TIME '00:00:00.000000'")
                .addRoundTrip("TIME '00:00:00.123456'", "TIME '00:00:00.123456'")
                .addRoundTrip("TIME '12:34:56'", "TIME '12:34:56'")
                .addRoundTrip("TIME '12:34:56.123456'", "TIME '12:34:56.123456'")

                // maximal value for a precision
                .addRoundTrip("TIME '23:59:59'", "TIME '23:59:59'")
                .addRoundTrip("TIME '23:59:59.9'", "TIME '23:59:59.9'")
                .addRoundTrip("TIME '23:59:59.99'", "TIME '23:59:59.99'")
                .addRoundTrip("TIME '23:59:59.999'", "TIME '23:59:59.999'")
                .addRoundTrip("TIME '23:59:59.9999'", "TIME '23:59:59.9999'")
                .addRoundTrip("TIME '23:59:59.99999'", "TIME '23:59:59.99999'")
                .addRoundTrip("TIME '23:59:59.999999'", "TIME '23:59:59.999999'")
                .addRoundTrip("TIME '23:59:59.9999999'", "TIME '23:59:59.9999999'")

                .execute(getQueryRunner(), trinoCreateAsSelect(getSession(), "test_time"))
                .execute(getQueryRunner(), trinoCreateAndInsert(getSession(), "test_time"));
```

so please remove comment (and optionally use the shorter form as above)

Reply:
Ah, I was trying this and it didn't work:
```
                .addRoundTrip("time(0)", "'00:00:00'", createTimeType(0), "TIME '00:00:00'")
```
I removed the comment and changed to short form, thanks!